content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-pomerium.adoc

```diff
@@ -296,3 +296,100 @@ For example:
 
 - You can build a policy that only allows users to access Jenkins at certain times of day or days of the week, or limit access to certain devices
 - You can import custom groups claims from your IdP and only allow access to members of the group
+
+== Configuring Pomerium with Unix Domain Sockets
+
+Starting from Jenkins version  2.452.1 , Jenkins supports Unix domain sockets. This can be particularly useful for improving security and performance.
+
+### Prerequisites
+
+Ensure that you have Pomerium installed and configured according to the [Pomerium documentation](https://www.pomerium.com/docs/).
+
+### Configuration Steps
+
+## 1. Update Pomerium Configuration :
+
+Add and update the following to your Pomerium configuration to enable Unix domain sockets:
+
+```yaml
+authenticate_service_url: https://authenticate.localhost.pomerium.io/oauth2/callback
+
+idp_provider: REPLACE_ME
+idp_provider_url: REPLACE_ME
+idp_client_id: REPLACE_ME
+idp_client_secret: REPLACE_ME
+
+signing_key: REPLACE_ME
+
+routes:
+  - from: https://verify.localhost.pomerium.io
+    to: http://verify:8000
+    pass_identity_headers: true
+    allow_websockets: true
+    policy:
+      - allow:
+          and:
+            - email:
+                is: user@example.com
+  - from: https://jenkins.localhost.pomerium.io
+    to: http://jenkins:8080
+    host_rewrite_header: true
+    pass_identity_headers: true
+    policy:
+      - allow:
+          and:
+            - domain:
+                is: example.com
+            - user:
+                is: username
+```
+
+## 2. Configure Jenkins :
```

Comment:
```suggestion
==== Configure Jenkins:
```
Adjusting formatting to be aligned with the rest of the page/site. Again, if these steps are all part of the Configuration Steps header, they do not need to be their own separate headings and could just be listed steps underneath "Configuration Steps".